### DISCUSSION TITLE
How to do custom notification template on UNRAID?

### DISCUSSION THREAD

**Discussion by AN1MATEK** [containrrr/watchtower#1703] [Q&A] [answered]
Hello there,

I cannot seem to make any template work in my UNRAID configuration. The thing is, that it is only possible to input one long line instead of breaking it down in many lines while using the GUI. If I just copy/paste the default template from [Watchtower's website](https://containrrr.dev/watchtower/notifications/), it gives this error at the start and the container stops:
```
time="2023-07-25T10:22:27+02:00" level=info msg="Watchtower 1.5.3"
time="2023-07-25T10:22:27+02:00" level=info msg="Using notifications: pushover"
time="2023-07-25T10:22:27+02:00" level=info msg="Checking all containers (except explicitly disabled with label)"
time="2023-07-25T10:22:27+02:00" level=info msg="Scheduling first run: 2023-07-25 12:22:27 +0200 CEST"
time="2023-07-25T10:22:27+02:00" level=info msg="Note that the first check will be performed in 1 hour, 59 minutes, 59 seconds"
time="2023-07-25T10:22:27+02:00" level=fatal msg="Notification template error" error="template: :1:7: executing \"\" at <.Report>: can't evaluate field Report in type []*logrus.Entry" notify=no
```

For reference, this is how the variable is configured:
![image](https://github.com/containrrr/watchtower/assets/15142596/26e9dfa2-f11f-4b3a-9d54-3f6a05f1f293)

I basically just want to tune the notification a bit, so instead of seeing a whole lot of info I can summarize it to "Watchtower successfully updated" or similar.

**Comment by piksel** [comment, marked as answer]
You are probably not passing the `--notification-report` flag to enable the new template format.

> Simple templates are used unless the notification-report flag is specified:
> `--notification-report` (env. `WATCHTOWER_NOTIFICATION_REPORT`): Use the session report as the notification template data.

A note about should probably be added to the `Report templates` section!

**Comment by AN1MATEK** [comment]
It was just that simple! Thanks a lot for the fix.

Just to add a comment here, if you are already going to edit the documentation: can you please mention that shoutrrr is built-in in watchtower? It took me quite a while to figure it out, I thought I needed to install shoutrrr and linked them somehow. Also while looking for an answer, I found other people with the same misconception. 

Just 2 cents.

Again, thanks a lot and have a great day.
